### PR TITLE
Replaced breadcrumb from react to web component

### PR DESCRIPTION
**Pull request by harshilmoradia**
## Summary
In an effort to standardize our components and benefit from the encapsulation and reusability that web components offer, we've transitioned our breadcrumb component from React to a web component.

- Fixed related cypress test for breaking changes.

## Related issue(s)
https://github.com/department-of-veterans-affairs/vets-design-system-documentation/issues/1632

## Testing done

Tested all the changes by navigating to to that specific URL in Chrom/Edge.

## Screenshots
Debt details:
Debt details

Disability benefits:
disability-benefits

GI:
gi

Medical copays:
medical-cppays

MHV:
MHV Mr Breadcrumb
MHV

Health care supply reorderning:
Screenshot 2023-11-07 at 12 19 29 PM

_Note: This field is mandatory for UI changes (non-component work should NOT have screenshots)._

|         | Before | After |
| ------- | ------ | ----- |
| Mobile  |        |       |
| Desktop |        |       |

## What areas of the site does it impact?

Only the Breadcrumbs are impacted for 11 instances.

## Acceptance criteria
- [ ] Deprecated component has either been removed entirely (if no longer in use) or moved to the next stage of the deprecation process.

### CODE REVIEW

**Review by Dr-Pongo — APPROVED** [review]
LGTM! 🚀

**Review by powellkerry — APPROVED** [review]

File: src/applications/claims-status/components/ClaimsBreadcrumbs.jsx [hunk 1]
```diff
@@ -27,7 +27,9 @@ class ClaimsBreadcrumbs extends React.Component {
 
   render() {
     return (
-      <Breadcrumbs>{this.renderBreadcrumbs(this.props.children)}</Breadcrumbs>
+      <VaBreadcrumbs>
+        {this.renderBreadcrumbs(this.props.children)}
```
Comment (jerekshoe) [inline src/applications/claims-status/components/ClaimsBreadcrumbs.jsx:31]: You should be able to use `va-breadcrumbs` here, no need for the `react-bindings` version

Comment (harshil1793) [reply src/applications/claims-status/components/ClaimsBreadcrumbs.jsx:30]: @jerekshoe fixed it.

**Review by jerekshoe — APPROVED** [review]
Looks good for `src/applications/claims-status`

File: src/applications/claims-status/components/ClaimsBreadcrumbs.jsx [hunk 2]
```diff
@@ -27,7 +26,9 @@ class ClaimsBreadcrumbs extends React.Component {
 
   render() {
     return (
-      <Breadcrumbs>{this.renderBreadcrumbs(this.props.children)}</Breadcrumbs>
+      <va-breadcrumbs>
+        {this.renderBreadcrumbs(this.props.children)}
```
Comment (mdewey) [inline src/applications/claims-status/components/ClaimsBreadcrumbs.jsx:30]: NITPICK: Super minor, but this is throwing a linter error. A non-blocker, but we should be looking to reduce the linting errors.

**Review by mdewey — APPROVED** [review]
LGTM! 

Minor NITPICK for a linting error, but good work updating to the new component!

**Review by KolbyVA — APPROVED** [review]

**Review by GovNapoleon — APPROVED** [review]

**Review by longmd — APPROVED** [review]

**Review by rmessina1010 — APPROVED** [review]
LGTM

**Review by randimays — APPROVED** [review]

**Review by Midge-dev — APPROVED** [review]
We own none of this but it looks good to me :call_me_hand:

**Pull request merged**